schema-loader/src/main/java/com/scalar/db/schemaloader/TableSchema.java

```diff
@@ -60,7 +60,7 @@ public TableSchema(String tableFullName, JsonObject tableDefinition, Map<String,
     String[] fullName = tableFullName.split("\\.", -1);
     if (fullName.length < 2) {
       throw new SchemaLoaderException(
-          "Parsing the schema JSON failed. Table full name must contains table name and namespace");
+          "parsing the schema JSON failed. The table full name must contains table name and namespace");
```

Comment:
I thought the style introduced by this PR was good until seeing this line, but I felt this message a bit weird since the second sentence is capitalized while the first sentence isn't capitalized. And on second thought, using the same convention for exception and log message might be simpler. This isn't a strong opinion, but what do you think? (cc: @brfrn169 @feeblefakie)

(If reverts or additional changes are needed, I can support you)

Reply:
Indeed, for this particular example, starting the message with a lowercase seems a bit weird. 
I don't necessarily have a strong opinion on it. Starting both exception and logs messages with a capital letter is fine with me.

Reply:
Just noticed we might want to share a error message string over log message, exception and/or gRPC response. In this case, it's hard to always follow the style?

```java
   String msg = "Something wrong happened";
   logger.warn(msg);
   responseObserver.onError(
          Status.INTERNAL.withDescription(msg).withCause(e).asRuntimeException());
```

```java
    catch (Exception e) {
       responseObserver.onError(
          Status.INTERNAL.withDescription(e.getMessage()).withCause(e).asRuntimeException());
    }
```

Reply:
@Torch3333 Okay, let's start both exception and log messages with a capital letter. Thanks.

Reply:
@komamitsu @brfrn169 Thank you for the suggestion. Let's update it.

@komamitsu 
> (If reverts or additional changes are needed, I can support you)
Thank you, I appreciate it. Let's discuss this offline.

Reply:
@Torch3333 Reverted the changes on the exception messages and added some changes
- https://github.com/scalar-labs/scalardb-sql/pull/241
- https://github.com/scalar-labs/scalardb-cluster/pull/143

Reply:
Thank you!

Reply:
Revised in https://github.com/scalar-labs/scalardb/pull/985/commits/a3067b2fc9e0b04aa74c001ec5f87f82eadeac90